src/pages/Scoreboard/Scoreboard.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+
+import ScoreBoard from "../../components/ScoreBoard/ScoreBoard";
+
+const ScoreBoardPage = () => {
+  return (
+    <div className="page page-score-board">
+      <section>
+        <ScoreBoard />
+      </section>
+    </div>
+  );
+}
+
+export default ScoreBoardPage;
```

Comment:
We can also apply our changes from the Player component here -- remove the curly body brackets and return but keep the parenthesis.